util/src/main/proto/Debug.proto

```diff
@@ -63,4 +63,40 @@ message Log {
     map<string, int64> level_summary = 1;
     optional CounterSet counter_set = 2;
     repeated Entry entries = 3;
+    repeated StatValidationResult stats_check_summary = 4;
+}
+
+message DataPoint {
+    message DataValue {
+        optional double value = 1;
+        repeated Log.Location locations = 2;
+    }
+
+    optional string date = 1;
+    repeated DataValue values = 2;
+}
+
+message StatValidationResult {
+    optional string place_dcid = 1;
+    optional string stat_var_dcid = 2;
+    optional string measurement_method = 3;
+    optional string observation_period = 4;
+    optional string scaling_factor = 5;
+    optional string unit = 6;
+
+    message StatValidationCounter {
```

Comment:
StatsValidationEntry?

This is the equivalent of Entry proto for stats. Should we use consistent field names?  counter_name -> counter_key, details -> user_message

Reply:
done. 

Updated counter_name to counter_key, but details is not equivalent to user_message. details is only set sometimes when needed and otherwise we jsut have the problem points. Maybe we can use additional_details